.github/CONTRIBUTING.md

```diff
@@ -1,15 +1,22 @@
-# How to Contribute
+# Contributing to Chapel
 
-As an open-source project, Chapel welcomes source contributions from 
-the community.  Chapel enforces the Developer Certificate of Origin (DCO) 
-on Pull Requests. It requires all commit messages to contain the 
-Signed-off-by line with an email address that matches the commit 
-author and the name on your GitHub account.
+As an open-source project, Chapel welcomes contributions from the
+community.  This document describes the high-level requirements for
+contributing source to Chapel, while other documents linked at the
```

Comment:
/source/source code/ ?

Reply:
I wrestled with wordings like this in a lot of places.  If you're contributing a change to a .rst or README file, is that source code?  I think I decided that "source" might be a broader category of what we accept.  Another approach would be to say "contributing changes" to Chapel, though of course if you're just contributing feedback or design thoughts on GitHub issues, the DCO isn't required...

I dunno...what do you think?

Reply:
Yeah, it's a bit ambiguous in either case. 

What do you think of calling it "changes to Chapel's git repository" or even more specifically "changes via pull request to Chapel's git repository" ?

Reply:
That sounds good to me, and I've adopted it.